.github/workflows/test_chart.yaml

```diff
@@ -59,7 +59,7 @@ jobs:
         run: kind load docker-image ${{ env.MANIFEST_IMG }}:${{ env.TAG }}
 
       - name: Run chart-testing (install)
-        run: helm install rancher-turtles out/charts/rancher-turtles/ -n rancher-turtles-system --create-namespace --wait --set cluster-api-operator.cert-manager.enabled=true
+        run: helm install rancher-turtles out/charts/rancher-turtles/ -n rancher-turtles-system --create-namespace --wait --set cluster-api-operator.cert-manager.enabled=true --set capi.enabled=false
```

Comment:
This workflow is the closest to quickstart process of the rancher turtles. Does this mean that capi operator will be disabled by default?

Reply:
Uninstalling doesn't really work for us right now properly due to CAPI operator not cleaning up providers before removing the controller. We can fix it on the CAPI operator side(preferable way) or document that users have to remove all providers manually before deleting the chart. 
I don't want CAPI operator blocking our progress on the extension so disabling CAPI is a temporary solution

Reply:
We are planning a v0.6.0 release for operator so fix can be included there